test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -18,30 +18,24 @@ Describe "Get-Content" -Tags "CI" {
     Remove-Item -Path $testPath -Force
     Remove-Item -Path $testPath2 -Force
   }
-  It "Should throw an error on a directory  " {# also tests that -erroraction SilentlyContinue will work.
-    Get-Content . -ErrorAction SilentlyContinue | Should Throw
-  }
-  It "Should return an Object when listing only a single line" {
-	  (Get-Content -Path $testPath).GetType().BaseType.Name | Should Be "Object"
-  }
-  It "Should deliver an array object when listing a file with multiple lines" {
-	  (Get-Content -Path $testPath2).GetType().BaseType.Name | Should Be "Array"
-  }
-  It "Should return the correct information from a file" {
-	  (Get-Content -Path $testPath) | Should Be $testString
-  }
-  It "Should be able to call using the gc alias" {
-	  { gc -Path $testPath } | Should Not Throw
-  }
-  It "Should be able to call using the type alias" {
-	  { type -Path $testPath } | Should Not Throw
-  }
-  It "Should return the same values for aliases" {
-	  $getContentAlias = Get-Content -Path $testPath
-	  $gcAlias         = gc -Path $testPath
-	  $typeAlias       = type -Path $testPath
-    $getContentAlias | Should Be $gcAlias
-    $getContentAlias | Should Be $typeAlias
+  It "Should throw an error on a directory  " {
+        try {
+            Get-Content . -ErrorAction Stop
+            throw "No Exception!"
+        }
+        catch {
+            $_.FullyQualifiedErrorId | should be "GetContentReaderUnauthorizedAccessError,Microsoft.PowerShell.Commands.GetContentCommand"
+        }
+  }
+  It "Should return an Object when listing only a single line and the correct information from a file" {
+        $content = (Get-Content -Path $testPath)
+        $content | Should Be $testString
+        $content.GetType().BaseType.Name | Should Be "Object"
```

Comment:
I believe that the test is similar to the following after it. So it is better to leave.